public/main.js

```diff
@@ -1,21 +1,28 @@
 //@ts-check
 import { Elm } from '../src/Main.elm';
 
-document.addEventListener("DOMContentLoaded", function () {
-  // Initial data passed to Elm (should match `Flags` defined in `Shared.elm`)
-  // https://guide.elm-lang.org/interop/flags.html
+// document.addEventListener("DOMContentLoaded", function () {
```

Comment:
Does this need to be commented out? if it's not needed it should be removed, not commented out.